<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>new_grammar</class>
 <widget class="QDialog" name="new_grammar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="next_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Következő</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Mégse</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="non_terminal_t">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>256</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="finish_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Kész</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="terminal_t">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>256</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="non_terminal_lab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nem-terminálisok</string>
   </property>
  </widget>
  <widget class="QLabel" name="terminal_lab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Terminálisok</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="nstart_t">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>104</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="s_lab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Kezdőjel</string>
   </property>
  </widget>
  <widget class="QLabel" name="helper_lab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>391</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Kérem adja meg  az adott jeleket szóközzel elválasztva!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Vissza</string>
   </property>
  </widget>
  <zorder>finish_button</zorder>
  <zorder>next_button</zorder>
  <zorder>cancel_button</zorder>
  <zorder>non_terminal_t</zorder>
  <zorder>terminal_t</zorder>
  <zorder>non_terminal_lab</zorder>
  <zorder>terminal_lab</zorder>
  <zorder>nstart_t</zorder>
  <zorder>s_lab</zorder>
  <zorder>helper_lab</zorder>
  <zorder>back_button</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
